script/testing/junit/traces/functions.test

```diff
@@ -540,5 +540,105 @@ SELECT split_part(str_a_val, 'd', 1) AS result FROM functions1 WHERE is_null = 1
 ----
 1 values hashing to 68b329da9893e34099c7d8ad5cb9c940
 
+query T nosort
+SELECT rpad(str_a_val, 0, str_i_val) AS result FROM functions1 WHERE is_null = 0
```

Comment:
Can you add tests that don't include the padding characters.

Reply:
Done